mixer/adapter/dogstatsd/config/config.proto

```diff
@@ -0,0 +1,43 @@
+syntax = "proto3";
+
+package adapter.dogstatsd.config;
+
+import "gogoproto/gogo.proto";
+
+option go_package="config";
+
+message Params {
+    // Address of the dogstatsd server, e.g. localhost:8125
+    string address = 1;
+
+    // prefix to prepend to all metrics
+    string prefix = 2;
+
+    // Maximum UDP packet size to send; if not specified defaults to 1432 bytes for better performance.
+    int32 buffer_length = 3;
+
+    // Tags to add to every metric
+    map<string, string> global_tags = 4;
+
+    // Only emit a percentage of metrics to the endpoint
+    double sample_rate = 5;
+
+    // Describes how to represent this metric in dogstatsd
+    message MetricInfo {
+        // Name of the metric in DataDog
+        string name = 1;
+        // Describes the type of metric
+        enum Type {
+            UNKNOWN = 0;
```

Comment:
nit: suggest naming this UNKNOWN_TYPE, per https://cloud.google.com/apis/design/naming_convention#enum_names